src/llama-vocab.cpp

```diff
@@ -1523,6 +1523,7 @@ void llama_vocab::impl::load(llama_model_loader & ml, const LLM_KV & kv) {
                     tokenizer_pre == "llama-v3" ||
                     tokenizer_pre == "llama-bpe"||
                     tokenizer_pre == "falcon3"  ||
+                    tokenizer_pre == "falcon_h1" ||
```

Comment:
The pre-tokenizer string should also use a dash instead of underscore, in order to be consistent with the reset of the identifiers:

```suggestion
                    tokenizer_pre == "falcon-h1" ||
```